tests/icons.test.js

```diff
@@ -25,8 +30,10 @@ icons.forEach(icon => {
     expect(subject.source).toEqual(icon.source);
   });
 
-  test(`${icon.title} has an "svg" value`, () => {
+  test(`${icon.title} has an "svg" value`, (done) => {
```

Comment:
It takes a long time because the `done` callback argument here. Jest is waiting for it to be called to finish the test: https://jestjs.io/docs/asynchronous#callbacks.

Reply:
You're right 👍🏼